src/styles/index.scss

```diff
@@ -1,3 +1,81 @@
+@import 'variables';
+
 body {
   margin: 0;
+  .block {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    height: 100vh;
+    .stopwatch {
+      width: $stopwatchDiam;
+      height: $stopwatchDiam;
+      border: 1vmin dotted $borderColor;
+      border-radius: 100%;
+      position: relative;
+      &__min-hand {
+        height: 20vmin;
+        width: 3vmin;
+        background-color: $blueColor;
+        position: absolute;
+        left: 50%;
+        transform: translate(-50%, 100%);
+        transform-origin: 50% 100%;
+        animation-name: minute-hand;
+        animation-duration: 3600s;
+        animation-timing-function: steps(60);
+        animation-iteration-count: infinite;
+      }
+      &__sec-hand {
+        height: 38vmin;
+        width: 1.5vmin;
+        background-color: $greenColor;
+        position: absolute;
+        transform: translate(-50%, 10px);
```

Comment:
The translate value '10px' in the transform property is a hard-coded value, which could cause inconsistency. Consider using a relative unit or a variable to maintain consistency across different screen sizes.